proto/core/io/address.proto

```diff
@@ -4,17 +4,18 @@ package syft.core.io;
 import "proto/core/io/location.proto";
 
 message Address {
-  string obj_type = 1;
+  string name = 1;
+  string obj_type = 2;
```

Comment:
I like the idea that Address has a name - although I'm concerned that it will be more than just for human-readable output. Aka - we shouldn't base any application logic off of name

Reply:
100% agree, we do not want people relying on these, but i have found during debugging when a node or client collapses into an Address it loses so much detail its hard to know what it is, if it retains a "name" and its 4x Locations its easy to figure it out even after serde and storage somewhere remotely.